llvm/test/Transforms/InstCombine/icmp-power2-and-icmp-shifted-mask.ll

```diff
@@ -448,7 +448,10 @@ define <2 x i1> @icmp_power2_and_icmp_shifted_mask_vector_swapped_splat_poison_2
 ; Vector of 2 reduction with splat containing undef
```

Comment:
The reason I didn't change these tests to use poison is that there are already existing test variants with poison, so I just kept both.